decko/features/step_definitions/wagn_steps.rb

```diff
@@ -87,7 +87,7 @@
   signed_in_as(username) do
     visit "/card/edit/#{cardname.to_name.url_key}"
     plusses.hashes.first.each do |name, content|
-      set_content "card[subcards][#{cardname}+#{name}][content]", content
+      set_content "card[subcards][+#{name}][content]", content
```

Comment:
Why this change? The old way is supposed to work too, no?

Reply:
Sorry, I'm wrong.  Yes, the old way would still work in a form.  But in this case, it means "set the content of the field named (X) to (Y).  Since the field now has a relative name, Capybara won't find the field without this change if we use the old name.